Classes_and_Objects/src/com/company/task_1/task_8/SosediCustomers.java

```diff
@@ -0,0 +1,25 @@
+package com.company.task_1.task_8;
+
+import java.util.ArrayList;
+
+public class SosediCustomers {
+    private final ArrayList<Customer> list;
+
+    public SosediCustomers(ArrayList<Customer> list) {
+        this.list = list;
+    }
+
+    public void addCustomer(Customer customer) {
+        this.list.add(new Customer(customer.id, customer.subName, customer.firstName, customer.patronymic, customer.address, customer.cardNumber,
```

Comment:
why do initializing Customer again? you've already passed it as an method param
just do `list.add(customer)`